system-x/services/horreum/src/main/java/software/tnb/horreum/validation/HorreumGenericException.java

```diff
@@ -0,0 +1,30 @@
+package software.tnb.horreum.validation;
+
+public class HorreumGenericException extends Exception {
```

Comment:
do you do something with this exception later in the test code? because from this PR it seems it is only wrapping the other exceptions and rethrowing